test/integration/docs/types.test.ts

```diff
@@ -0,0 +1,65 @@
+import * as fs from 'node:fs/promises';
```

Comment:
Doesn't the mkdocs build check that and inform about dead links?
Most of the time that I fixed links was because the build informed about them...